### PR TITLE
Make sure we handle both getter and setter for scrollTop etc

### PR DESCRIPTION
**Pull request by arv**
Previously we did not forward the setter which lead to issues.

Fixes #272

### CODE REVIEW

**Comment by arv** [comment]
@jmesserly, @azakus

File: src/wrappers/HTMLElement.js [hunk 1]
```diff
@@ -130,11 +130,15 @@
...
 
-  function getterRequiresRendering(name) {
-    defineGetter(HTMLElement, name, function() {
+  function getter(name) {
+    return function() {
       scope.renderAllPending();
       return this.impl[name];
-     });
+    };
+   }
```
Comment (jmesserly) [inline src/wrappers/HTMLElement.js]: is this off by one space?

**Comment by jmesserly** [comment]
LGTM, thank you!

File: src/wrappers/HTMLElement.js (hunk 1 above)
Comment (arv) [inline src/wrappers/HTMLElement.js]: Yeah, this looks suspicious... rebasing and testing again

Comment (arv) [inline src/wrappers/HTMLElement.js]: Oh, wrong indentation... I thought it was a merge error.

**Pull request merged**